src/main/webapp/index.html

```diff
@@ -19,26 +19,72 @@
   <head>
     <meta charset="UTF-8">
     <title>CodeU 2019 Starter Project</title>
-    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="css\main.css">
+    <link rel="stylesheet" href="css\index.css">
     <script src="/js/navigation-loader.js"></script>
   </head>
   <body onload="addLoginOrLogoutLinkToNavigation();">
     <nav>
       <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-        <li><a href="/community.html">Community</a></li>
-        <li><a href="/stats.html">Stats</a></li>
-        <li><a href="/chart.html">Charts</a></li>
-        <li><a href="/favouriteplaces.html">Favourite Places</a></li>
-
+        <li><a href="/"><img src="images/home-logo.png" height="25px"></a></li>
+        <li><a href="/">PUBLIC FEED</a></li>
```

Comment:
"PUBLIC FEED" and "COMMUNITY" are in uppercase whereas "Your Page" and "Logout" are normally cased.

Reply:
Good point!